yogstation.dme

```diff
@@ -1599,6 +1599,7 @@
 #include "code\modules\clothing\gloves\_gloves.dm"
 #include "code\modules\clothing\gloves\boxing.dm"
 #include "code\modules\clothing\gloves\color.dm"
+#include "code\modules\clothing\gloves\knifeboxing.dm"
```

Comment:
Can you just discard the changes on this file?

Reply:
It makes it alphabetical, like it should be.